src/components/ContextProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { Store } from "redux";
+import buildStore from "../store";
+import { State } from "../reducers/index";
+
+export interface ContextProviderProps extends React.Props<any> {
+  csrfToken: string;
```

Comment:
is it okay for this to still be a required prop when you're not passing it down from the server? i see you commented out the other uses in this file.

Reply:
It doesn't seem to be breaking anything.  (But in any event: I'm currently working on a branch for implementing login; that will get rid of all the unnecessary CSRF stuff!)